src/schemas/EthDenverCredential/1-0-0.ts

```diff
@@ -0,0 +1,24 @@
+import { JSONSchema7 } from "json-schema";
+import BaseVerifiableCredential from "../BaseVerifiableCredential/1-0-0";
+
+export default {
+  ...BaseVerifiableCredential,
+
+  $schema: "http://json-schema.org/draft-07/schema#",
+  $id: "https://raw.githubusercontent.com/discoxyz/disco-schemas/main/json/EthDenverCredential/1-0-0.json",
+
+  title: "Disco @ EthDenver 2023",
```

Comment:
My Mac didn't recognize the disco ball emoji (probably because I don't have the latest one) so we can omit this for now.